dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/environment/production/db/migration/hsqldb/V0_5__gaen_exposed_keys.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Created by Ubique Innovation AG
+ * https://www.ubique.ch
+ * Copyright (c) 2020. All rights reserved.
```

Comment:
Nitpick//Observation/Question: `All rights reserved.` Is this an outdated copyright headers?
#75 looks like it replaced the `All rights reserved.` with `MPL-2.0`
https://github.com/DP-3T/dp3t-sdk-backend/pull/75/files
_(This applies to other files of this PR, but I only make one comment - maybe there is a good reason for this header)_